tests/FakeItEasy.Analyzer.VisualBasic.Tests/UnusedReturnValueAnalyzerTests.cs

```diff
@@ -351,7 +351,7 @@ Namespace TheNamespace
     Class TheClass
         Sub Test()
             Dim foo = New Fake(Of IFoo)()
-            A.CallTo(foo).Where(Function() True, New Object())

```

Comment:
Had to fix this after enabling `Option Strict`. Yes, VB is perfectly happy to accept `New Object()` where an `Action<IOutputWriter>` is expected... How anyone could think it would be a good idea is beyond me 😕